src/Service/OS2Forms/CaseSubmissionNormalizers/CaseSubmissionNormalizer.php

```diff
@@ -118,7 +108,7 @@ protected function getConfig(): array
                         throw new WebformSubmissionException('blabla '.$submissionData['naevn']);
                     }
 
-                    $complaintCategory = $entityManager->getRepository(ComplaintCategory::class)->findOneByNameAndBoard($submissionData[$spec['os2FormsKey']] ?? null, $board);
+                    $complaintCategory = $entityManager->getRepository(ComplaintCategory::class)->findOneByNameAndBoard($submissionData[$spec['os2forms_key']] ?? null, $board);
 
                     if (null === $complaintCategory) {
                         throw new WebformSubmissionException('blabla');
```

Comment:
Maybe we could come up with a better exception message.

Reply:
Oops. Fixed.